### PR TITLE
MEPTS-1422 | LISTA DE DAH DIFF FGH

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/advancedhivillness/ListOfPatientsOnAdvancedHivIllnessQueries.java [hunk 1]
```diff
@@ -151,11 +151,43 @@ public String getCd4ResultBellowOneYear() {
...
+        + "                      ON e.encounter_id = o.encounter_id  "
+        + "INNER JOIN ( "
+        + getLastCd4ResultDateQueries()
+        + " ) last_cd4 ON last_cd4.person_id = ps.person_id  "
+        + " WHERE  ps.voided = 0  "
+        + "  AND e.voided = 0   "
+        + "  AND o.voided = 0  "
+        + "  AND e.encounter_type IN ( ${6}, ${13}, ${51} )  "
+        + "  AND o.concept_id = ${1695}  "
+        + "  AND o.value_numeric < 200  "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/advancedhivillness/ListOfPatientsOnAdvancedHivIllnessQueries.java]: same as the other ticket, please use parameters and if possible use the same method

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
Please check

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/advancedhivillness/ListOfPatientsOnAdvancedHivIllnessQueries.java (hunk 1 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/advancedhivillness/ListOfPatientsOnAdvancedHivIllnessQueries.java]: Implemented

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsInAdvancedHivIllness.java [hunk 2]
```diff
@@ -16,8 +16,9 @@
 import org.openmrs.module.reporting.report.ReportDesign;
 import org.openmrs.module.reporting.report.definition.ReportDefinition;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 
-@Deprecated
+@Component
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsInAdvancedHivIllness.java]: Why?

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@helderjosue please see

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsInAdvancedHivIllness.java (hunk 2 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupListOfPatientsInAdvancedHivIllness.java]: corrected

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**